docker/Dockerfile

```diff
@@ -0,0 +1,42 @@
+# Dockerfile to setup neo-python on the master branch.
+# https://github.com/CityOfZion/neo-python/tree/master/docker
+#
+# Building (these commands create a Docker image called neopython):
+#
+#    $ docker build -f Dockerfile -t neopython .
+#
+# Build without caching:
+#
+#    $ docker build --no-cache -f Dockerfile -t neopython .
+#
+# At this point there is a Docker image called 'neopython'. If you want to run it connecting
+# to a private network, make sure the privatenet container is already running.
+# https://hub.docker.com/r/cityofzion/neo-privatenet
+#
+# Start a container interactively, opening a bash in `/neo-python`, and mounting the current directory as `/neo-python/sc`:
+#
+#    $ docker run --rm -it -v $(pwd):/neo-python/sc --net=host -h neopython --name neopython neopython /bin/bash
+#
+# Once you are inside the container, you can start neo-python with `python3 prompt.py -p` (using -p to connect to a private net).
+# To update neo-python, just run `git pull` and `pip install -e .`
+FROM ubuntu:16.04
```

Comment:
Why not use `python:3.5` as the base image? 

Benefits:

- Smaller image size
- No need to install Python dependencies
- Faster image build time

Reply:
The reason is that the Python image need more space! I did test the builds for both `python:3.5` and `ubuntu:16.04` and the Python image uses 200MB more compared to Ubuntu.

Reply:
Ignore me, I thought it was the other way round 😂